src/components/MapLayerList/LayerGroup.js

```diff
@@ -38,3 +38,41 @@ Props
   - list of layers in group
   - Not sure yet
 */
+
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Accordion from '@mui/material/Accordion';
+import Layer from './Layer';
+
+export default function LayerGroup(props) {
+  const { chartLayerList } = props;
+  const subHeadings = [];
+  const subListings = {};
+  chartLayerList.forEach((layer) => {
+    if (!subHeadings.includes(layer.ChartInputSubHeading)) {
+      subHeadings.push(layer.ChartInputSubHeading);
+      subListings[layer.ChartInputSubHeading] = [layer];
+    } else {
+      subListings[layer.ChartInputSubHeading].push(layer);
+    }
+  });
+  const buildSubListings = (subHeading) => (
+    subListings[subHeading].map((layer) => <Layer key={layer.id} lData={layer} />)
+  );
+  const buildSubGroup = (subHeading) => (
```

Comment:
this is something I always go back and forth on. do you make this a separate component?  I am open to the discussion but tend to feel it should be separate. otherwise, a component can be filed with these and it's hard to find (and re-use)